### PR TITLE
Chore/38 add component scaffolding tool

### LINKED ISSUE #38: [CHORE] add component scaffolding tool for time savings
Instead of creating new files and folders and typing boilerplate, devs optionally can type a command in the terminal and save time (and typos)

### PR DESCRIPTION
**Pull request by mmnoo**
# Description

Make it possible to save time with a cli command instead of typing new component boilerplate

Closes #38 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] chore: dev support tooling

# Instructions

See readme in this diff

# How Has This Been Tested?

ran `plop test` and a folder appears in the components directory with the expected folders, files and code.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: mrtt-ui/README.md [hunk 1]
```diff
@@ -1,46 +1,5 @@
...
-
-### `npm run build`
-
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
```
Comment (JBurkinshaw) [inline mrtt-ui/README.md:30]: Can we keep some of this as it is still relevant? In the future we can update and modify as required.
I plan to add more specific info to the `gmw-users` root README.md as as part of https://github.com/globalmangrovewatch/gmw-users/issues/40 as well

Comment (julia-rom) [reply mrtt-ui/README.md:30]: agreed! otherwise plop sounds great hah

**Review by julia-rom — CHANGES_REQUESTED** [review]

File: mrtt-ui/README.md (hunk 1 above)
Comment (mmnoo) [reply mrtt-ui/README.md:30]: sure.

**Review by JBurkinshaw — APPROVED** [review]

**Comment by julia-rom** [comment]
would like to approve this! stuck on requested re-review like yesterday cc @mmnoo

**Pull request merged**